Contributing.md

```diff
@@ -0,0 +1,33 @@
+
+# Contributing 👬️
+
+Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+
+When contributing to this repository, please first discuss the change you wish to make via issue, 
+email, or any other method with the owners of this repository before making a change. 
+
+
+
+## Pull Request Process 🔻️
+
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b feature/feaute_name`)
+3. Commit your Changes (`git commit -m 'Add some feature'`)
+4. Update the README.md file , with description about the feature  **\#optional**
+5. Push to the Branch (`git push origin feature/AmazingFeature`)
+6. Open a Pull Request
+
+**Always remember when you are stuck at somewhere, you can always reach out we are ready to help**
+
+## Contact 📬️
+
+👨‍💻️ Your name - [@twitter_handle](https://twitter.com/twitter_handle)
+
+✉️ Email - <a href="mailto:yourname ">samplemail@gmail.com</a>
+
+🏗️ Project Link: [https://github.com/devcer/CapShun](https://github.com/devcer/CapShun)
+
```

Comment:
Need to update this with details of any of the owner

Reply:
twitter handle should be good. @isantoshv - https://twitter.com/isantoshv
You can remove the email.
I can merge this after the changes.